### PR TITLE
Adds the 'Antiseptic Saliva' trait

### PR DESCRIPTION
**Pull request by CameronWoof**
**What it does**
Grants access to the 'Lick Wounds' verb, which will target an adjacent creature and bandage/salve any wounds they have on the targeted limb, at the cost of nutrition. 

**Okay, but why?**
I only recently dove into the meaning behind the phrase 'lick your wounds' and finding out that some animals have spit with antiseptic properties really interested me. In addition, I thought that heal bellies could use a non-vore alternative for those without an interest in it.

**This should be a positive trait**
This is balanced against heal bellies, which are better (soon to be _much_ better) and free for anyone to make and use. It didn't make sense to give this a cost considering there was a free and more effective alternative. Since it can only be used on others, I'd expect this to see sporadic use by people with character motivations that align with it.

### CODE REVIEW

**Comment by Wickedtemp** [comment]
I CAN LITERALLY MLEM PEOPLE AND IT IMPROVES THEIR HEALTH

**Comment by Novacat** [comment]
Should this be available on vanilla species?

**Comment by CameronWoof** [comment]
I think it'd fit especially well on Tajaran, but I'd like to hold that for its own follow-up PR. My experience with touching roundstart race balance on other servers makes me a little gun-shy about rolling that in with this PR.

**Comment by SpadesNeil** [comment]
This sounds broken and overpowered.

But it's vore so like I give a fuck.

File: code/modules/mob/living/carbon/human/species/station/traits_vr/neutral.dm [hunk 1]
```diff
@@ -113,3 +113,12 @@
 	..(S,H)
 	H.verbs |= /mob/living/proc/glow_toggle
 	H.verbs |= /mob/living/proc/glow_color
+
+/datum/trait/antiseptic_saliva
+	name = "Antiseptic Saliva"
+	desc = "Your saliva has especially strong antiseptic properties that can be used to heal small wounds."
+	cost = 0
```
Comment (SpadesNeil) [inline code/modules/mob/living/carbon/human/species/station/traits_vr/neutral.dm]: Probably should make it cost at least 1.

Comment (CameronWoof) [reply code/modules/mob/living/carbon/human/species/station/traits_vr/neutral.dm]: The logic for this is given in the PR itself--it doesn't make sense to me to charge for this when heal bellies are free and better.

**Comment by Novacat** [comment]
Not as overpowered as you think, bandages/ointment are the easiest medical supplies to get on the Tether. Though, thinking about it, the verb should probably check if the target is wearing voidsuits or any other types of clothing that would normally restrict bandaging/salving.

**Comment by CameronWoof** [comment]
@Novacat It does.

**Comment by Wickedtemp** [comment]
Even without the buff PR, healbelly is much, much more powerful. And free. So it kinda fits that a less powerful thing in the same vein would be free.

**Comment by Wickedtemp** [comment]
Bandage/Disinfect are two things the healbelly doesn't actually do, but they're only used for Brute or Burn injuries.

Currently, thats all the healbelly actually fixes, and it does so faster than any non-cryo reagent in Medical, unless you start making mixes. Quickly enough that you can heal someone to nearly full health in about a minute. The new PR reduces the healing value for brute and burn a bit but still remains substantially better and faster overall than the lick wounds.

**Comment by Heroman3003** [comment]
The one thing I do agree on is that I think it should be a 1-cost positive trait. It is not overpowered at all in my opinion. In fact, I'd say it also injecting a very miniscule amount of Numbing Enzyme (like 30 ticks worth of it at most) if there are actually non-treated wounds would make it perfectly good for a one-cost trait.

**Comment by Heroman3003** [comment]
Not everything vore absolutely must have a free equivalent available for non-vore. And as I said, make it slightly better in its own way (numbing thing, for example) and it would be good for 1-cost trait. All current 0-cost traits are PURELY scene (Brutal predation, Bloodsucker, Succubus drain), visual (Autohisses, Glowing eyes/body), or completely fluff-based minor changes with equal amount of upsides and downsides (Metabolisms, Hot/Cold blood).

One neutral trait I'd say can in any way count as always-positive is Trash Can, but that is truly a negligible positive with nutrition from trash really being barely anything unless in huge amounts.
Antiseptic saliva on the other hand, offers while not better-than-existing options, still significant positive. 


On an unrelated note, whatever happens, xenochimerae should definitely have it by default, if it gets added at all.

**Comment by CameronWoof** [comment]
The purpose of this trait is to provide a non-vore alternative to heal bellies. I'm pretty opposed to punishing people who aren't into vore by making this cost trait points.

**Comment by Heroman3003** [comment]
Its not punishing people who aren't into vore in any way that I see it. Having a vorish alternative to healing is what heal-belly was meant to be, which is why its always less effective than actual medical help. Way to make healing into a scene. So unless its 'punishing players who only want mechanical aspect of thing and dont want thing to be vore', then yes, it is punishing, and I think it SHOULD be punished. And players that aren't into vore _at all_ is not that relevant point at all IMO.

**Comment by CameronWoof** [comment]
This was designed and balanced for downstream. The idea is that people who would be tempted to use heal bellies for mechanical benefit would use this instead, to bring the vore/non-vore options up to parity. I don't think that liking vore should confer a mechanical advantage the way that heal bellies do, and while this IS significantly worse, it's much better than nothing. 

I think that it is penalizing people who aren't into vore when you give vore free abilities and advantages that others either don't have access to, or charge them for it.

**Comment by Heroman3003** [comment]
Vore option even existed as alternative in and of itself. It was always an alternative to regular treatment, awful mechanically, but vorish. That was the point. Way to integrate vore into game. It WAS alternative, and now you're saying because there are niche situations where is good and can be used, non-vore players should get their personal free alternative to alternative that was made to encourage more vore. One trait point is not that much, and adding a directly positive trait into Neutral traits, especially with non-insignificant benefit is an awful idea. The healbelly is not a big advantage unless you abuse it as purely-mechanical thing. And regarding penalizing players that are not into vore at all, I will always go. So what? There is already entire medical system. That was there in the first place and to which healbelly is vorish alternative. THAT is their free alternative to healbelly.

**Comment by CameronWoof** [comment]
We clearly feel differently about this, but I'm not interested in making the trait cost points unless I'm told in no uncertain terms that this won't be merged otherwise. I just don't agree with that philosophy.

**Comment by Heroman3003** [comment]
I mean, I like the trait, and I think this is great example of what positive traits COULD be like, not just stat sticks. But I really dislike idea of adding stuff purely because non-vore people are jealous of vore features.

**Comment by Wickedtemp** [comment]
I of course have absolutely zero bias in the fact that mlemming people may become a viable medical strategy.

**Comment by Sebbe9123** [comment]
Make it a cost 1 at least, there is no real reason not to as it is buff to whoever takes it, be it a "vore alternative" or not.

**Comment by Screemonster** [comment]
Turning every player (at least, every custom species player) into a walking medbay at no point cost is a bit much. The whole defining point of customspecies was supposed to be that they were flexible but not strictly as "good" in pure numbers terms as the base species. ~~And then everyone immediately leapt upon it to powercreep them.~~

**Comment by CameronWoof** [comment]
Scree, that's literally already the case with heal bellies. All this does is give a similar ability to people who don't want to participate in vore.

**Comment by Sebbe9123** [comment]
> Scree, that's literally already the case with heal bellies. All this does is give a similar ability to people who don't want to participate in vore.

I don't understand this, we have this already, it's called "medkits" they usually contain proper medical equipment for problems like these that people who don't like vore can always, and always has, used. 
Making medical redundant to people who do Vore and only needed in extreme cases for people who don't like vore seems like... I feel like powercreep doesn't quite cover it, it's more like "why have medical supplies at all if you can go on a hunting spree, get your buddy to lick your wounds and shove you in a belly and never need medical. Honestly might as well just automate Medical instead of having it at all at that point.

**Comment by CameronWoof** [comment]
Heal bellies are a way to channel nutrition into healing. The trade-off is that rather than having physical supplies, you use nutrition as the resource. It is free, anyone can do it, and it will soon be far more effective than it already is.

This trait is also a way to channel nutrition into healing. The trade-off is that rather than having physical supplies, you use nutrition as the resource. It is free, anyone can do it, and it is far less effective than heal bellies are.

If you see a problem with one and not the other, I do not know what to tell you. Our design philosophies are simply incompatible.

**Comment by Wickedtemp** [comment]
> > Scree, that's literally already the case with heal bellies. All this does is give a similar ability to people who don't want to participate in vore.
> 
> I don't understand this, we have this already, it's called "medkits" they usually contain proper medical equipment for problems like these that people who don't like vore can always, and always has, used.
> Making medical redundant to people who do Vore and only needed in extreme cases for people who don't like vore seems like... I feel like powercreep doesn't quite cover it, it's more like "why have medical supplies at all if you can go on a hunting spree, get your buddy to lick your wounds and shove you in a belly and never need medical. Honestly might as well just automate Medical instead of having it at all at that point.

Because that isn't the case. If what you're describing is accurate - "First aid + healbelly = Medical is useless otherwise", then we'd already be seeing this happen ingame. Even with the improvements being made to the healbelly to make it a little more viable, it's not going to be, nor will it ever be, the end-all-be-all in medical. I know this because the healbelly used to be better, around the strength it's getting buffed to, and it was actually rarely used. Every now and then we'd have a patient brought in by somebody else via healbelly, I can honestly only recall that happening one specific time, Verin brought somebody in through maintenance into the back of Medical using the healbelly. And this was years ago. We still used the NSS Exodus Bay map.

Since then, I think I've used it once or twice just because I had to transport a bunch of patients, primarily wolpin, through a hazardous area. 

Healbelly was never used as a replacement for medical treatment, and let's be honest, if your injuries could be fixed by a first-aid kit, you don't absolutely *need* Medical. It helps to have them take a look at you and confirm you're all okay, but if you know your friend was hit once or twice and a quick lick will suffice, honestly, that isn't a whole lot I'm missing out on as Medical anyway. Consider the inverse. They don't do it, and they call Medical. I run down there, give them a bit of gauze and ointment and then go back to whatever it was I was doing before. People heal themselves with first-aid rather frequently, especially Sec, Mining, and Science, and it doesn't really affect Medical that much. They all have access to first aid kits, it's not a huge deal if those kits get used. 

Now, granted, this is a bit different. This is a character ability that lets you bandage/salve wounds without gauze or ointment, at the cost of nutrition. It's kinda vore-ish because it's licking, but also not exactly considered to be vore, because it's not *actually* vore. I don't think this is an issue, either, because as of now... The healbelly, without the improvements it's likely getting in another PR, is actually *much, much better* than what this ability adds. The healbelly, right now, heals brute and burn faster than Medical can, unless they use Cryo. And you can use it anywhere. With hardly any nutrition usage. It heals the damage so quickly that, as long as you swallow someone quickly, they don't actually even need first-aid, because the current healbelly heals all of the brute/burn in like two minutes. You could have someone in crit from being mauled by a Mimic, and if you're already there and you eat them, throw on heal, there's actually a decent chance you'll have nearly all of the brute damage healed by the time medical arrives. 

That's what we have now, and that's what we've had for... well, since we've had the current iteration of the Vore Panel I think. 

And it isn't a problem. It isn't over-used, in fact I rarely actually ever see it used at all. Medical already has obscenely powerful tools that can literally heal almost everything outside of shrapnel and bone fractures - the Cryocells - but they aren't always relied upon, either. The majority of medical players, from my experience, just have fun doing medical. Using the kits, chems, the stasis bags and defibs, etc, to help their patients. I doubt this will have a huge negative impact, I'd be okay with it costing a trait point, but I personally don't see it as a necessity. 

In the end, I don't really even see this as a "non-vore alternative to healbelly", because that isn't entirely accurate. Healbelly does more than this. This is just something that's kinda silly and fun and all it does it act as a basic 'aid kit', which... I don't really care about. It's first-aid. If all it takes to fix you is first-aid, Medical isn't missing out on much if you treat it yourself. Just...make sure you know how to treat it correctly.

**Comment by Wickedtemp** [comment]
Nah, the saliva trait is the same as the basic white kits, and worse than the red kits that medical has access to. 

The first aid kit would actually be better because it also has an injector and analayzer, allowing you to do more.

**Comment by CameronWoof** [comment]
I don't expect to see this trait used overly much. It can't be used on yourself, which ought to stop the more selfish players, but beyond that, we have corporate regulations against patting people who don't want to be touched--I imagine getting licked by someone would be similar. 

What I _expect_ is that primarily medical players will use this trait on people they're close to as a more friendly and intimate way to heal their wounds, and that some non-medical players may take it to kiss booboos in non-emergency situations. 

I personally don't plan for it to replace my usual gauze-and-ointment grab on my Security character, just because licking total strangers would be weird, at least to my own character's sensibilities.

**Comment by Wickedtemp** [comment]
Yeah, that's another point.

ICly, you wouldn't go around licking random people just because they're hurt. Especially if that licking counts as medical treatment. Cause if they didn't want you to do that, that's battery. If you lick them and they didn't want to be licked, potential battery. If they refuse medical treatment and you force it, also ICly against regs and SOP. This would kinda be both.

Cameron pretty much nailed how I plan on using this. Everyone Tempest likes -and likes hir back-  is getting tongue-baths instead of gauze most of the time now.

**Comment by Sebbe9123** [comment]
You need to look away from medical being in the focus, which this PR isn't or if it is this PR should not call itself an alternative to healbellies ,  and see how this affects the playerbase on the whole. 

Most of our playerbase is a custom species, our playerbase has a tendency to power game or meta game which is undeniable but considering what type of server we are a certain amount of power gaming is fine.

If this gets introduced with prefbreak and other rules such as SOP, there should be a case for making it a mechanical pref too since we already have the lick verb.

Medical will see little use of this considering its only good for stabilisation and unless med is already there someone will do it before them.

I see Explorers and Miners getting more use out of this than med, even if security decided to go Emo.

And lastly you said yourself that healbellies are worse than what they seemingly attempt to simulate, why should special slobber be a direct buff to white first aid kits.

**Comment by CameronWoof** [comment]
This isn't a direct buff over white first-aid kits. A white first-aid kit gives you an inaprovaline shot and a health scanner, neither of which can be simulated with this. In addition, you'd absolutely starve trying to heal as many injuries as a white kit gives you unless you stocked up on nutrition beforehand. While it's true that buying a protein shake is easier than getting a white kit, those tasks exist on the same basic field of effort. If you really wanted one or the other, it would not be difficult.

And yes, I still consider this an alternative to heal bellies and expect they'd be used in similar circumstances--willing and consenting friends (though I readily admit a lick is much less intimate than vore) sharing an intimate moment after one of them is injured. While this certainly won't be the _only_ way it is used, I think that IC consequences can shape things towards a pretty sane middle ground.

**Comment by Haery70** [comment]
Personally, I'm gonna have to go with the 'no' camp on this one. It may work on downstreams, but not here to be the alternative to healbellies for players who don't want to participate in vore because, personally, I'd rather allow the healbelly ability to be a vore-exclusive ability, especially if this trait is going to be free. It defeats the purpose of the server and could alienate the vore and non-vore camps further. Just because vore players have it doesn't mean that an equivalent should be made for those who don't want to participate in what we do here.

As for the cost of the trait, if this was to become a thing, I'd go with the 'Make it a 1' camp on this one, maybe even 2. I don't really care about the whole 'penalizing/punishing players who don't want to participate in vore' argument since, from what I've seen, they're not the target audience. If players want to patch others up without the use of vore they can use a medkit for all I care, but making this a neutral trait is going to have a lot of powergamers in the more action-oriented roles take up the trait and run around like a mobile medbay as Scree has stated. And yes, I don't care about the whole 'They do this with healbellies all the time' argument because, again, this is a vore server where people come here to get ate or eat people. There's already regret by some of the players for the traits system being abused by powergamers which the system was made before we had stuff like expeditions, so why put more kindle into that fire? And if this trait does end up making it with no cost, then I'd be more happy to see a Whitelist for traits being made to prevent such abuse.

tl;dr: No support for the idea and it should remain a vore-exclusive ability on here, but can work on the downstreams. But, then again, I'm just a guy who plays on the main server making a comment on here. So, it's up to the actual devs to decide on what to do with this.

**Comment by CameronWoof** [comment]
For the most part this is just a philosophical disagreement, but I did want to say, if your worry is 'further alienating the vore-nonvore camps' then you should want them to be brought closer to parity. Jealously guarding your vore-specific superpowers is going to cause a lot more ire than letting everyone enjoy them in their own way.

**Comment by SpadesNeil** [comment]
Either it costs a point or we don't add it.
image

Maybe two if we find it to be OP.

**Comment by CameronWoof** [comment]
Understandable. I'm not interested in punishing people for not enjoying a fetish, so I won't be hosting the PR that does it myself. Someone else is more than welcome to take the code and open up their own PR, with or without credit, if they find it worthwhile.

**Comment by Screemonster** [comment]
I don't understand why this was a trait rather than just adding it to licks for everyone, since effectively it provided a mechanical benefit at zero cost to custom-races, and non-custom races lose out because they can't take traits.
